addons/web/static/src/js/framework/misc.js

```diff
@@ -172,6 +172,29 @@ function ReloadContext (parent, action) {
 }
 core.action_registry.add("reload_context", ReloadContext);
 
+/**
+ * This client action simply displays a notification. It is useful when we
+ * do an action (such as a server action), that does not need to do anything,
+ * except give some feedback to the user.
+ * @param {ActionManager} parent
+ * @param {Object} action
+ * 
```

Comment:
what we did elsewhere in master is to document sub parameters like this:

* @param {string} action.title
* @param {string} [action.message]
* @param {boolean} [action.sticky=false]

Note that optional parameters are in brackets, with optional value if it makes sense